Islandora-CLAW/islandora_demo
# Stop unneccessary events

**GitHub Issue**: https://github.com/Islandora-CLAW/CLAW/issues/982

# What does this Pull Request do?

Touches up some config to prevent unneccessary events from getting fired.

# What's new?

Mostly just requiring all conditions to be met in the derivative contexts.  Also the external uri for the image tag wasn't exported properly, so that's included now.  I also noticed PID was showing up out of place so I tucked it away.

# How should this be tested?

- Make an object and confirm unnecessary events are being sent (I like to throw a dsm here, though you can also watch Karaf logs).
- Pull in this PR
- `drush fim -y islandora_demo`
- Make another object. You shouldn't have unneccessary events.


# Interested parties
@Islandora-CLAW/committers
